jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/jsonp/JsonpDiagnosticCollector.java

```diff
@@ -32,7 +42,40 @@ public void completeDiagnostic(Diagnostic diagnostic) {
 
     @Override
     public void collectDiagnostics(ICompilationUnit unit, List<Diagnostic> diagnostics) {
+        if (unit == null) {
+            return;
+        }
+        List<MethodInvocation> allMethodInvocations = ASTUtils.getMethodInvocations(unit);
+        List<MethodInvocation> createPointerInvocations = allMethodInvocations.stream()
+                .filter(this::isCreatePointerInvocation).collect(Collectors.toList());
+        for (MethodInvocation m: createPointerInvocations) {
+            Expression arg = (Expression) m.arguments().get(0);
+            if (isInvalidArgument(arg)) {
+                String msg = "createPointer target must be a sequence of '/' prefixed tokens or an emtpy String";
```

Comment:
This msg should be placed in the constants file with a constant like CREATE_POINTER_ERROR_MESSAGE

Reply:
Fixed